### PR TITLE
Consume geojson from s3

### PR DESCRIPTION
**Pull request by jeffgrunewald**
Fixes the failure to retry the topic creation for data ingestions.

Also fixes flakey tests related to application env setting download_dir and conflicting with the setting in the`config/test.exs` (removed).

### CODE REVIEW

**Comment by jeffgrunewald** [comment]
For anyone who commented already; I added a 100 ms sleep between the topic create call and the validation check. Since the retry was in there originally assuming the broker hadn't necessarily created the topic _immediately_ this felt like a sane amount of time to wait without holding up creation too long.
The topic creation is idempotent, but this would theoretically prevent unnecessarily spamming the broker with creation requests.

File: lib/reaper/data_extract/processor.ex [hunk 1]
```diff
@@ -91,9 +91,9 @@ defmodule Reaper.DataExtract.Processor do
   end
 
   defp create_topic(topic) do
```
Comment (LtChae) [inline lib/reaper/data_extract/processor.ex]: I'd love to see us move this create stuff farther upstream into a blocking `init` so these flow through synchronously in the overall `Init`. Until then, though this should do - Ben

Comment (jeffgrunewald) [reply lib/reaper/data_extract/processor.ex]: this is now an exponential backoff @LtChae and @bbrewer-papa-x

Comment (jeffgrunewald) [reply lib/reaper/data_extract/processor.ex]: stream of randomized delays increasing roughly 10% each time, no single one ever to exceed 2 seconds and the entire attempt never to exceed 30 seconds

**Pull request merged**